includes/admin/class-wc-rest-upe-flag-toggle-controller.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Class WC_REST_Upe_Flag_Toggle_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for UPE feature flag.
+ */
+class WC_REST_Upe_Flag_Toggle_Controller extends WP_REST_Controller {
```

Comment:
Based on `WC_REST_` being uppercase, it seems that this should be:

```suggestion
class WC_REST_UPE_Flag_Toggle_Controller extends WP_REST_Controller {
```

Reply:
Since it's literally the same name I updated it in 01c8df38 😄